3Dmol/parsers.js

```diff
@@ -2202,5 +2202,52 @@ $3Dmol.Parsers = (function() {
 	}
         return atoms;
     }
+
+    /**
+     * Parse a lammps trajectory file from str and create atoms
+     */
+    parsers.lammpstrj = parsers.LAMMPSTRJ = function(str, options){
```

Comment:
Isn't this a trajectory file (ie, it needs a topology file to be complete)?  If so it should be in GLModel.setCoordinates, right?